apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common/ErrorReporting/ReportEventsTo.cs

```diff
@@ -22,7 +22,7 @@ namespace Google.Cloud.Diagnostics.Common
     /// <summary>
     /// The location error events will be sent to.
     /// </summary>
-    public enum ReportEventsToLocation
+    public enum EventTarget
```

Comment:
No, I don't feel too strongly about it. I do like the fluent syntax of something like `ReportEventsTo.Logging(...)` but in other contexts the class name feels a bit awkward. I don't have a better idea though. If @jskeet is ok with it, I am too.